File: app/src/main/java/com/jinjoo/pouch/model/Cosmetic.java
package com.jinjoo.pouch.model;

public class Cosmetic {

    String img;
    String name;
    String brand;

    @Override
    public String toString() {
        return "Cosmetic{" +
                "img='" + img + '\'' +
                ", name='" + name + '\'' +
                ", brand='" + brand + '\'' +
                '}';
    }

    public Cosmetic(String name, String brand) {
        this.name = name;
        this.brand = brand;
    }

    public String getImg() {
        return img;
    }

    public void setImg(String img) {
        this.img = img;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }
}

File: app/src/main/kotlin/com/jinjoo/pouch/api/model/Cosmetic.kt
package com.jinjoo.pouch.api.model

class Cosmetic(var name: String, var brand: String) {
    var img: String? = null
    override fun toString(): String {
        return "Cosmetic{" +
                "img='" + img + '\'' +
                ", name='" + name + '\'' +
                ", brand='" + brand + '\'' +
                '}'
    }

}